test/app/views/application/content_for.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render ContentForComponent do |comp| %>
+  <p>Have a great day.</p>
+  <% comp.content_for(:title) do %>
```

Comment:
As we discussed on our call, this line could be:

```suggestion
  <% component.with(:title) do %>
```

Reply:
done